lib/cb/utils/api.rb

```diff
@@ -57,10 +57,14 @@ def cb_delete(path, options = {}, &block)
       def execute_http_request(http_method, uri, path, options = {}, &block)
```

Comment:
Just wanted to throw that out there as a thing to ponder and perhaps spend some time on - I wouldn't be heartbroken if it merged the way it is now.